lib/extras/packed_image.h

```diff
@@ -74,6 +83,52 @@ class PackedImage {
     }
   }
 
+  float GetPixelValue(size_t y, size_t x, size_t c) const {
+    const uint8_t* data = const_pixels(y, x, c);
+    switch (format.data_type) {
+      case JXL_TYPE_UINT8:
+        return data[0] * (1.0f / 255);
+      case JXL_TYPE_UINT16: {
+        uint16_t val;
+        memcpy(&val, data, 2);
+        return swap_endianness_ ? JXL_BSWAP16(val) : val;
```

Comment:
@szabadka, is this missing a `* (1.0f / 65535)`?

Reply:
Thanks for catching this! I sent PR #2820 to fix.